ui/jobs/buff_tracker.ts

```diff
@@ -561,8 +589,12 @@ export class BuffTracker {
         }
 
         const map = propToMapMap[prop];
-        map[key] = map[key] || [];
-        map[key]?.push(buff);
+        if (Array.isArray(key)) {
+          key.forEach((k) => map[k] = [buff, ...map[k] ?? []]);
+        } else {
+          map[key] = map[key] || [];
```

Comment:
```suggestion
          map[key] ??= [];
```

Reply:
Dang!
I didn't even know there is such an operator make it much clear!